src/components/ProbeEditor.tsx

```diff
@@ -20,22 +21,33 @@ import { LabelField } from 'components/LabelField';
 import ProbeStatus from './ProbeStatus';
 import { InstanceContext } from 'contexts/InstanceContext';
 import { trackEvent, trackException } from 'analytics';
+import { GrafanaTheme2 } from '@grafana/data';
+import { Clipboard } from 'components/Clipboard';
```

Comment:
Q: do we have clipboard in grafana ui, if not it think it makes sense for us to add it in grafana ui/toolkit.

we have other places(integrations) in Grafana where copy to clipboard is used.

Reply:
Yeah, that's a good point. I adapted this one from how they were doing it in integrations, but it'd take some thought on how to make this more generic I think.